### ISSUE TITLE
Remove Button error if there is input with id="id"

### ISSUE THREAD

**Issue by shyevsa** [unclead/yii2-multiple-input#162]
The error is 
```
jquery.js:1468 Uncaught Error: Syntax error, unrecognized expression: #[object HTMLInputElement]
    at Function.Sizzle.error (jquery.js:1468)
    at Sizzle.tokenize (jquery.js:2125)
    at Sizzle.select (jquery.js:2546)
    at Function.Sizzle [as find] (jquery.js:869)
    at jQuery.fn.init.find (jquery.js:2792)
    at jQuery.fn.init (jquery.js:2909)
    at jQuery (jquery.js:75)
    at removeAttribute (jquery.multipleInput.js:389)
    at HTMLInputElement.<anonymous> (jquery.multipleInput.js:313)
    at Function.each (jquery.js:365)
```
if somewhere in the page there is an input with `id='id'`

```
<input type="text" name="id" id="id" class="form-control" >
```

**Comment by unclead** [comment]
PR is always welcome

**Comment by unclead** [comment]
fixed ad43b3d8062a4011ae2c4a7bc69215e2b4c49282
